docu/release_notes/2016-12-06-Technev153.html.md

```diff
@@ -0,0 +1,15 @@
+---
+date: December 20, 2016
+service: 'hybris techne'
+official_version: v1.5.3
+headline: 'Avatars and Images Pattern'
+---
+
+### New Pattern
+* New pattern that that displays avatars and images on different context. Markup and iteraction on how to Upload/Change images.
+  ![Tree screenshot](img/v1.5.3/avatars-images.png)
+
+### Minor updates
+* Simplified **Tree Pattern* HTML Markup. Removed complexity with more explicit CSS classes.
+* On the **Toogle Buttons** we now show a stylised "1/0" instead of "on/off". More universal now;  no need for translations.
+* Improved accesability on the **expandable button**. The expandable options are now in a more accesible order.
```

Comment:
[You might provide a "For example," here. Which options, and what makes the new order more accessible?]
+* The expandable button options are now in an improved, more accessible order.